integration-tests/kubernetes/pom.xml

```diff
@@ -1,21 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <artifactId>quarkus-integration-tests-parent</artifactId>
-    <groupId>io.quarkus</groupId>
-    <version>999-SNAPSHOT</version>
-  </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
 
-  <artifactId>quarkus-integration-test-kubernetes-parent</artifactId>
-  <name>Quarkus - Integration Tests - Kubernetes - Parent</name>
-  <packaging>pom</packaging>
-
-  <modules>
-    <module>maven-invoker-way</module>
-    <module>quarkus-standard-way</module>
-    <module>quarkus-standard-way-kafka</module>
-  </modules>
+    <artifactId>quarkus-integration-test-kubernetes-parent</artifactId>
+    <name>Quarkus - Integration Tests - Kubernetes - Parent</name>
+    <packaging>pom</packaging>
 
+    <profiles>
+        <profile>
+            <id>test-kubernetes</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
```

Comment:
FWIW, I personally try to avoid `activeByDefault` because I usually don't like it's auto-deactivate semantics:
https://maven.apache.org/guides/introduction/introduction-to-profiles.html#explicit-profile-activation
> This profile will automatically be active for all builds unless another profile in the same POM is activated using one of the previously described methods. All profiles that are active by default are automatically deactivated when a profile in the POM is activated on the command line or through its activation config.

I'm usually doing something like that instead: https://github.com/quarkiverse/quarkus-cxf/commit/002d812231a8e1d70bd10f3c50af89e89bd27343#diff-38cc9162c1bf9934153e28bed107952752e909c08cc52bfd310871b7379f45e6R122-R127

YMMV!